compiler/ann-api/include/NeuralNetworks.h

```diff
@@ -230,6 +230,40 @@ typedef enum {
+     * * 1: A 4-D tensor, of shape [depth_out, filter_height, filter_width, depth_in],
+     *      specifying the filter. For tensor of type ANEURALNETWORKS_TENSOR_QUANT8_SYMM_PER_CHANNEL
+     *      the channel dimension (ANeuralNetworksSymmPerChannelQuantParams::channelDim) must be
+     *      set to 0.
+     * * 2: A 1-D tensor, of shape [depth_out], specifying the bias. For input tensor of type
+     *      ANEURALNETWORKS_TENSOR_FLOAT32 or ANEURALNETWORKS_TENSOR_FLOAT16 the bias must be
+     *      of the same type. For filter tensor of ANEURALNETWORKS_TENSOR_QUANT8_ASYMM and
+     *      ANEURALNETWORKS_TENSOR_QUANT8_ASYMM_SIGNED, the bias should be of ANEURALNETWORKS_TENSOR_INT32,
+     *      with zeroPoint of 0 and bias_scale == input_scale * filter_scale. For filter tensor of
+     *      ANEURALNETWORKS_TENSOR_QUANT8_SYMM_PER_CHANNEL, the bias should be of ANEURALNETWORKS_TENSOR_INT32,
+     *      with zeroPoint of 0 and bias_scale of 0. The actual scale of each value 'i' is equal to
+     *      bias_scale[i] = input_scale * filter_scale[i].
+     * * 3: An ANEURALNETWORKS_INT32 scalar, specifying the padding on the left, in the ‘width’ dimension.
+     * * 4: An ANEURALNETWORKS_INT32 scalar, specifying the padding on the right, in the ‘width’ dimension.
+     * * 5: An ANEURALNETWORKS_INT32 scalar, specifying the padding on the top, in the ‘height’ dimension.
+     * * 6: An ANEURALNETWORKS_INT32 scalar, specifying the padding on the bottom, in the ‘height’ dimension.
+     * * 7: An ANEURALNETWORKS_INT32 scalar, specifying the stride when walking through input in the ‘width’ dimension.
+     * * 8: An ANEURALNETWORKS_INT32 scalar, specifying the stride when walking through input in the ‘height’ dimension.
+     * * 9: An ANEURALNETWORKS_INT32 scalar, and has to be one of the FuseCode values. Specifies the activation to
+     *      invoke on the result.
+     * * 10: An optional ANEURALNETWORKS_BOOL scalar, default to false. Set to true to specify NCHW data layout for
+     *       input0 and output0. Available since API level 29.
+     * * 11: An optional ANEURALNETWORKS_INT32 scalar, specifying the dilation factor for width. Defaults to 1.
+     *       If set to k > 1, there will be k-1 skipped cells between each filter element on width dimension.
+     *       If this input is set, input 12 (dilation factor for height) must be specified as well. Available since
+     *       API level 29.
+     * * 12: An optional ANEURALNETWORKS_INT32 scalar, specifying the dilation factor for height. Defaults to 1.
+     *       If set to k > 1, there will be k-1 skipped cells between each filter element on height dimension.
+     *       If this input is set, input 11 (dilation factor for width) must be specified as well. Available since
+     *       API level 29.
+     * 
+     * Inputs (explicit padding):
+     * * 0: A 4-D tensor, of shape [batches, height, width, depth_in], specifying the input.
```

Comment:
@YiHyunJin Please update `runtime/nnapi-header/include/NeuralNetworks.h` instead of `ann-api/include/NeuralNetworks.h`.